lib/hammer_cli_foreman/ssh.rb

```diff
@@ -36,10 +37,16 @@ def execute
         return HammerCLI::EX_OK
       end
 
+      if concurrent.to_i == 0
+        warn _("specify 1 or more concurrent hosts")
+        return HammerCLI::EX_OK
+      end
+
       ssh_options = { :user => user, :auth_methods => ['publickey'] }
       ssh_options[:keys] = [identity_file] unless identity_file.to_s.empty?
 
-      Net::SSH::Multi.start(:on_error => :warn) do |session|
+      Net::SSH::Multi.start(:concurrent_connections => concurrent.to_i, :on_error => :warn) do |session|
```

Comment:
It looks like the assumption I made here was that concurrent.to_i would return nil if concurrent was set to nil. It actually returns 0. This makes my statements in regards to line 15 somewhat non-sensical. I will update the pull request to have the behavior I intended.